recipes/valijson/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def requirements(self):
+        self.requires("rapidjson/cci.20200410")
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
+            schema_file = os.path.abspath(os.path.join(self.source_folder, "schema.json"))
+            valid_file = os.path.abspath(os.path.join(self.source_folder, "valid.json"))
+            invalid_file = os.path.abspath(os.path.join(self.source_folder, "invalid.json"))
```

Comment:
Could this be embedded into the cpp?

Reply:
I tried to create an example as close to the usage section of valijson, as possible.
They do expect files to be passed to parse jsons from.
That's why I'm passing it here like this.

Reply:
That's fair, I prefer to having less extra support files. This is just a suggestion :smile: